### ISSUE TITLE
Netcore2.0 Unix Socket Tcp Send Big Buffer Raise Error Message too long

### ISSUE THREAD

**Issue by caozhiyuan** [dotnet/corefx#23579]
https://github.com/caozhiyuan/DotNetty/tree/dev/examples/SocketAsyncClient
https://github.com/caozhiyuan/DotNetty/tree/dev/examples/SocketAsyncServer
when nioBuffers total byte size big than 344064,will raise 10040 MessageSize Error（Message Too Long）

```c#
SocketError errorCode;
long localWrittenBytes = this.clientSocket.Send(nioBuffers, SocketFlags.None, out errorCode);
if (errorCode != SocketError.Success && errorCode != SocketError.WouldBlock)
{
    throw new SocketException((int)errorCode);
}
```

**Comment by taylorreece** [comment]
Was this issue actually solved in 2.1.0, or was it closed prematurely?  My team started running into this today with an 8MB message raising a "Message too long" error. Our app is dotnet core 2.0.0 on a Unix box.

**Comment by caozhiyuan** [comment]
2.0.3 fix  it
